packages/cerebral/typings/index.d.ts

```diff
@@ -0,0 +1,109 @@
+// <reference types="node" />
+// <reference types="function-tree" />
+
+
+declare module 'cerebral/devtools' {
```

Comment:
That's not that simple. Unless I put index.d.ts in our root folder (/packages/cerebral/), TS compiler doesn't recognize it as 'cerebral' module and doesn't allow `import { Controller } from "cerebral";`. 

So if I move index.d.ts to src folder (in order to work with /devtools/ subfolder) the compilation breaks.

Also, another problem will come with inferno and react 'submodules' inside viewFactories folder, since they are not folders, so I'm not sure how to place d.ts files there.

Help! :)